### ISSUE TITLE
Bug: Quassel Webserver crashes if the webpage is open for a long time

### ISSUE THREAD

**Issue by eparon** [magne4000/quassel-webserver#165]
After connecting to Quassel Webserver's page and logging in using my core's credentials. if i leave this page open in a tab for a long time the webserver crashes.

The error I get to the STDERR of my system is:
```_http_outgoing.js:346
    throw new Error('Can\'t set headers after they are sent.');
    ^

Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:346:11)
    at ServerResponse.header (/home/user/quassel-webserver/node_modules/express/lib/response.js:719:10)
    at ServerResponse.send (/home/user/quassel-webserver/node_modules/express/lib/response.js:164:12)
    at ServerResponse.json (/home/user/quassel-webserver/node_modules/express/lib/response.js:250:15)
    at ServerResponse.send (/home/user/quassel-webserver/node_modules/express/lib/response.js:152:21)
    at Socket.<anonymous> (/home/user/quassel-webserver/node_modules/net-browserify-alt/api.js:152:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Socket.writeAfterFIN [as write] (net.js:288:8)
    at WebSocket.<anonymous> (/home/user/quassel-webserver/node_modules/net-browserify-alt/api.js:180:20)
```
---------------------------
node.js version: 6.3.0
quassel-core version: 0.12.4

**Comment by magne4000** [comment]
Nice catch. Fixed by `net-browerify-alt@0.6.3`, so `npm update` and it's all good.

**Comment by eparon** [comment]
nice, that seems to have fixed this problem!

however, i encountered a new issue and the following error was logged:
```
/home/user/quassel-webserver/node_modules/ws/lib/Receiver.js:386
    default: srcBuffer.copy(dstBuffer, dstOffset, 0, length); break;
                       ^

RangeError: out of range index
    at RangeError (native)
    at fastCopy (/home/user/quassel-webserver/node_modules/ws/lib/Receiver.js:38
6:24)
    at Receiver.add (/home/user/quassel-webserver/node_modules/ws/lib/Receiver.j
s:86:3)
    at TLSSocket.realHandler (/home/user/quassel-webserver/node_modules/ws/lib/W
ebSocket.js:800:20)
    at emitOne (events.js:96:13)
    at TLSSocket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at TLSSocket.Readable.push (_stream_readable.js:135:10)
    at TLSWrap.onread (net.js:542:20)
```
